src/burn/engine/exeengine.cpp

```diff
@@ -583,8 +731,10 @@ extern "C" HRESULT ExeEngineExecutePackage(
     ReleaseHandle(pi.hThread);
     ReleaseHandle(pi.hProcess);
 
-    // Best effort to clear the execute package cache folder variable.
+    // TODO: Figure out if we care about "clearing" BURN_BUNDLE_EXECUTE_PACKAGE_ACTION (= 0)
+    // Best effort to clear the execute package cache folder and action variables.
     VariableSetString(pVariables, BURN_BUNDLE_EXECUTE_PACKAGE_CACHE_FOLDER, NULL, TRUE);
+    VariableSetNumeric(pVariables, BURN_BUNDLE_EXECUTE_PACKAGE_ACTION, BOOTSTRAPPER_ACTION_STATE_NONE, TRUE);
```

Comment:
Smells weird to switch the type of the variable out from under people, though. I thought about just leaving it. Time for https://twitter.com/barnson/status/588811755507347457?